db/migrate/20170825180550_add_waiver_field_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddWaiverFieldToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :read_and_accepted_waiver_form, :boolean, default: false
```

Comment:
How is this different from Terms and Condition?

Reply:
After last meeting with Hanan we agreed that terms and conditions can be present at the footer of the website and not part of signing up. The waiver form will be replacing terms and conditions. 

The reason we need a new column because we have to make old users who agreed to Term and Conditions read the waiver form and agree to it.